source/components.tsx

```diff
@@ -167,3 +168,16 @@ export const Calendar: EC<CalendarProps> = () => <></>;
 Calendar.optionOf = optionCreator('calendar');
 
 Calendar.loadModule = componentLoader(['CalendarComponent']);
+
+
+export type AxisPointerProps = EventHandlerProps & AxisPointerComponentOption;
+/**
+ * @example
+ * ```tsx
+ * <AxisPointer link={[{ xAxisIndex: 'all' }]} />
```

Comment:
```suggestion
 * ```tsx
 * <AxisPointer link={[{ xAxisIndex: 'all' }]} />
 * ```
```